db/migrations/versions/3a22407701c8_.py

```diff
@@ -18,25 +18,24 @@
 
 def upgrade():
     # ### commands auto generated by Alembic - please adjust! ###
-
+    with op.get_context().autocommit_block():
+        op.execute("ALTER TYPE flagtype ADD VALUE 'RESOLVED'")
     op.drop_column("flags","resolution_reason")
     op.execute("DROP TYPE resolutiontype;")
```

Comment:
LGTM, as long as no-one is using these yet.

Reply:
OKay great, ive tested with resolution data on downgrades and upgrades and was happy so hopefully this wont cause issues either way (the column is nullable)